src/main/java/bobby/command/Command.java

```diff
@@ -0,0 +1,159 @@
+package bobby.command;
+
+import bobby.manager.FileManager;
+import bobby.exception.IncorrectDescriptionFormatException;
+import bobby.exception.NoDescriptionException;
+import bobby.manager.ResponseManager;
+import bobby.manager.TaskManager;
+import bobby.task.Deadline;
+import bobby.task.Event;
+import bobby.task.Task;
+import bobby.task.ToDo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Command {
+    private String command;
+    private TaskManager taskManager;
+    private ArrayList<Task> taskList;
+
+    public Command(String command, TaskManager taskManager) {
+        this.command = command;
+        this.taskManager = taskManager;
+        this.taskList = taskManager.getTaskList();
+    }
+
+    public String getCommand() {
```

Comment:
As the name of this class is Command, the getCommand method name is a bit confusing. Maybe getCommandText?